onnxscript/rewriter/generic_pattern.py

```diff
@@ -1093,136 +844,73 @@ def make_rule(self) -> orp.RewriteRule:
         return GenericRewriteRule(self)
 
 
-class OnnxGenericPattern(GenericPattern):
-    """An instance of GenericPattern taking onnx model.
-
-    It defines the matching pattern and its replacement.
-
-    :param match_proto: the onnx function defining the matching pattern
-    :param apply_proto: the onnx function defining the new pattern
-    :param validate_mapping: the function used to validate a pattern
-    :param use_onnxscript: tells if the apply_proto is an onnxscript function or
-        a regular function returning a FunctionProto
-    :param opsets: opset to consider when converting the function into ONNX,
-        if not specified, it is opset 18 for the main opset, and opset 1
-        for domain com.microsoft.
-    :param verbose: in [0, 10], increase the verbosity to understand why a pattern
-        does not match
-    """
-
+class FunctionPattern(GenericPattern):
```

Comment:
What's this for? What's the difference between FunctionPattern and GenericPattern?

Reply:
It was `OnnxGenericPattern`. It's a feature supporting Callable and FunctionProto to become Pattern. I will add the description.

Reply:
Done